deploy/docker/production/nginx.conf

```diff
@@ -80,13 +100,33 @@ http {
         location / {
             # Normalize duplicate slashes
             if ($request ~ ^(GET|HEAD)\s([^?]*)//(.*)\sHTTP/[0-9.]+$) {
-                return 301 $2/$3;
+                rewrite "^" https://$host$uri permanent;
             }
             proxy_set_header  Host  $http_host;
             proxy_set_header  X-Forwarded-For    $proxy_add_x_forwarded_for;
-            proxy_set_header  X-Forwarded-Proto  $scheme;
-            proxy_pass  http://app;
             proxy_set_header  Connection  "";
+
+            # "ifs are evil" except for when you need them
+            # the below code is a "hack" for doing "if cond1 AND cond2" - Will Oct 14 21
+            # if we are a cgap*.hms.harvard.edu host
+            set $is_https_host "";
+            if ($host ~* (cgap.*\.hms\.harvard\.edu)) {
```

Comment:
Do you now want to add a hyphen after cgap to disallow the original `cgap.hms.harvard.edu`? Are we going to be using that name again? (Maybe if not we should leave it up with forwarding info.) Or is it only `fourfront-cgap` we are discontinuing?